### PR TITLE
Source Hubspot: fix acceptance tests

### PR DESCRIPTION
**Pull request by roman-yermilov-gl**
## What
Disable CAT state validation test due to failing tests. The source have complex conflicts with the latest CDK version, so the new test is temporarily disabled as a workaround. Reduced time of entire sync to avoid timeouts during acceptance test run

### CODE REVIEW

**Review by tolik0 — APPROVED** [review]

File: airbyte-integrations/connectors/source-hubspot/metadata.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 36c891d9-4bd9-43ac-bad2-10e12756272c
-  dockerImageTag: 4.0.0
+  dockerImageTag: 4.0.1
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-hubspot/metadata.yaml:13]: Please set this to `4.1.0` due to `airbyte-cdk = "^0"` change

**Review by askarpets — CHANGES_REQUESTED** [review]

**Review by askarpets — APPROVED** [review]

**Review by brianjlai — COMMENTED** [review]
> Disable CAT state validation test due to failing tests. The source have complex conflicts with the latest CDK version, so the new test is temporarily disabled as a workaround. Reduced time of entire sync to avoid timeouts during acceptance test run

I'm curious to dig into this a little bit. Was there a specific version of the CDK was posing a problem? I noticed we are bumping using `"^0"` so it sounds like we're not concerned about pinning to a specific version. So presumably `poetry lock` was run back when the CDK package was only at `0.73.0`. Can we rerun this command and push the change to the lockfile to the PR?

Which part of the `validate_state_messages` was failing? Was it the verifying the recordCounts field was on the state message, or the requirement that every stream emit at least one final state message.

**Comment by roman-yermilov-gl** [comment]
> > Disable CAT state validation test due to failing tests. The source have complex conflicts with the latest CDK version, so the new test is temporarily disabled as a workaround. Reduced time of entire sync to avoid timeouts during acceptance test run
> 
> I'm curious to dig into this a little bit. Was there a specific version of the CDK was posing a problem? I noticed we are bumping using `"^0"` so it sounds like we're not concerned about pinning to a specific version. So presumably `poetry lock` was run back when the CDK package was only at `0.73.0`. Can we rerun this command and push the change to the lockfile to the PR?
> 
> Which part of the `validate_state_messages` was failing? Was it the verifying the recordCounts field was on the state message, or the requirement that every stream emit at least one final state message.

Update: made `poetry lock` and deleted `validate_state_messages`. In order to incremental tests to pass I have also prepared PR: https://github.com/airbytehq/airbyte/pull/36814 with fix of the test itself

**Comment by lazebnyi** [comment]
follow up - https://github.com/airbytehq/airbyte/pull/36814

**Pull request merged**